mule-extensions-api/src/test/java/org/mule/runtime/extension/api/test/mimetype/FileExtensionContentTypeResolutionTestCase.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.test.mimetype;
+
+import static org.mule.runtime.api.metadata.MediaType.BINARY;
+import static org.mule.runtime.api.metadata.MediaType.HTML;
+import static org.mule.runtime.api.metadata.MediaType.JSON;
+import static org.mule.runtime.api.metadata.MediaType.TEXT;
+import static org.mule.runtime.api.metadata.MediaType.XML;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.mule.runtime.api.metadata.MediaType;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.activation.MimetypesFileTypeMap;
+
+import org.junit.Test;
+
+/**
+ * Ensures that some of the defined mimeTypes are properly mapped. This test does not cover all the available mimeType mappings as
+ * that will require to parse the mime.types file
+ */
+public class FileExtensionContentTypeResolutionTestCase {
```

Comment:
Shouldn't this test be part of the new `mule-extensions-mime-types` module? I'm just guessing by the package name. Or is it because the new module is not JPMS-modularized yet?

Reply:
yes, but this is also testing that the mime types config can be read form another module (which is the issue being addressed by moving this), will add that as a comment.